lib/misc/cells.coffee

```diff
@@ -0,0 +1,32 @@
+scopes = require './scopes'
+{values} = require 'underscore-plus'
+
+module.exports =
+
+  getCurrentCell: (ed) ->
+    buffer = ed.getBuffer()
+    start = buffer.getFirstPosition()
+    end = buffer.getEndPosition()
+    regexString = atom.config.get 'julia-client.cellDelimiter'
+
+    unless regexString?
+      return [start, end]
```

Comment:
This won't work -- you're not returning an object with `range` and so on as it's fields. Not that it matters, since this will never be called anyways.